core/src/main/java/org/springframework/security/authentication/DelegatingReactiveAuthenticationManager.java

```diff
@@ -37,6 +40,20 @@ public class DelegatingReactiveAuthenticationManager implements ReactiveAuthenti
 
 	private final List<ReactiveAuthenticationManager> delegates;
 
+	private boolean continueOnError = false;
+
+	private static final Log logger = LogFactory.getLog(DelegatingReactiveAuthenticationManager.class);
+
+	public DelegatingReactiveAuthenticationManager(boolean continueOnError, List<ReactiveAuthenticationManager> delegates) {
```

Comment:
Since `continueOnError` is an optional parameter, will you please make it a setter instead? This will bring it inline with other Spring Security components.

Reply:
Done.